src/shared/utils/vis.ts

```diff
@@ -241,32 +241,40 @@ export const defaultYColumn = (
   return null
 }
 
-export const mosaicYColumn = (
+export const defaultYSeriesColumns = (
   table: Table,
-  preferredColumnKey?: string
-): string | null => {
-  const validColumnKeys = getStringColumns(table)
-  if (validColumnKeys.includes(preferredColumnKey)) {
-    return preferredColumnKey
+  preferredYSeriesColumns: Array<string>
+): Array<string> => {
+  const validColumnKeys = [...getStringColumns(table)]
+  let ySeriesColumns = []
+
+  if (Array.isArray(preferredYSeriesColumns)) {
```

Comment:
if `preferredYSeriesColumns` is required, is this check necessary?

Reply:
Yes, because even TypeScript allows null